doc/devices/ahs_remote.rst

```diff
@@ -0,0 +1,100 @@
+The remote AHS device
+=====================
+
+The remote AHS device of the PennyLane-Braket plugin runs Analogue Hamiltonian Simulation (AHS) on
+Amazon Braket's remote service. AHS is a quantum computing paradigm different from gate-based computing.
+AHS uses a well-controlled quantum system and tunes its parameters to mimic the dynamics of another quantum
+system, the one we aim to study.
+
+The remote service provides access to running AHS on hardware. As AHS devices are not gate-based, they are not
+compatible with the standard PennyLane operators. Instead, they are compatible with `pulse programming <https://docs.pennylane.ai/en/stable/code/qml_pulse.html>`_ in PennyLane.
+
+More information about AHS and the capabilities of the hardware can be found in the `Amazon Braket Developer Guide <https://docs.aws.amazon.com/braket/latest/developerguide/braket-devices.html#braket-qpu-partner-quera>`_.
+
+Usage
+~~~~~
+
+After the Braket SDK and the plugin are installed, and once you
+`sign up for Amazon Braket <https://docs.aws.amazon.com/braket/latest/developerguide/braket-enable-overview.html>`_,
+you have access to the remote AHS device in PennyLane.
+
+Instantiate an AWS device that communicates with the hardware like this:
+
+>>> import pennylane as qml
+>>> s3 = ("my-bucket", "my-prefix")
+>>> remote_device = qml.device("braket.aws.ahs", device_arn="arn:aws:braket:us-east-1::device/qpu/quera/Aquila", s3_destination_folder=s3, wires=3)
+
+This device can be used with a QNode within PennyLane. It accepts circuits with a single ``ParametrizedEvolution``
+operator based on a hardware-compatible ``ParametrizedHamiltonian``. More information about creating PennyLane operators for AHS can be
+found in the `PennyLane docs <https://docs.pennylane.ai/en/stable/code/qml_pulse.html>`_.
+
+Creating a register
+^^^^^^^^^^^^^^^^^^^
+
+The atom register defines where the atoms will be located, and determines the strength of the interaction
+between the atoms. Here we define coordinates for the atoms to be placed at (in micrometers), and create a constant
+interaction term for the Hamiltonian:
+
+.. code-block:: python
+
+    coordinates = [[0, 0], [0, 5], [5, 0]]  # number of coordinate pairs must match number of device wires
+
+    H_interaction = qml.pulse.rydberg_interaction(coordinates)
+
+Creating a global drive
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Hardware currently supports a single global drive pulse applied to all atoms in the register.
```

Comment:
I was not sure whether this means "a single global drive and other, local drives" or "a single drive that must act globally".

Reply:
I think it's clearer now that Korbinian's suggestion for wording has been applied!